install/share/profiles/acmeServerCert.cfg

```diff
@@ -0,0 +1,107 @@
+profileId=acmeServerCert
+classId=caEnrollImpl
+desc=ACME profile for use in FreeIPA deployments
```

Comment:
I think I'd drop the Free bit and make it just for IPA deployments for better cross-distro undestanding.

Reply:
Fixed in next push.